src/avram/query_builder.cr

```diff
@@ -110,11 +118,19 @@ class Avram::QueryBuilder
     @distinct || @distinct_on
   end
 
+  def limit
+    @limit
+  end
```

Comment:
This (as well as the offset) could probably just be `getter` macros, but for consistency sake, I just added them as methods.